Description: Export IncompleteRead from requests.compat since it's
 imported by python-pip.
Author: Daniele Tricoli <eriol@mornie.org>
Forwarded: not-needed
Bug-Debian: https://bugs.debian.org/766419
Last-Update: 2014-10-23

--- a/requests/compat.py
+++ b/requests/compat.py
@@ -92,6 +92,7 @@
     from Cookie import Morsel
     from StringIO import StringIO
     from urllib3.packages.ordered_dict import OrderedDict
+    from httplib import IncompleteRead
 
     builtin_str = str
     bytes = str
@@ -107,6 +108,7 @@
     from http.cookies import Morsel
     from io import StringIO
     from collections import OrderedDict
+    from http.client import IncompleteRead
 
     builtin_str = str
     str = str
